ddR/NAMESPACE

```diff
@@ -40,7 +39,7 @@ export(is.dlist)
 export(is.dobject)
 export(is.sparse_darray)
 export(nparts)
-export(parallel)
+export(parallel.ddR)
```

Comment:
Right now there is a redundancy when using `library()` to load a backend package and then `useBackend()` to actually enable it. Maybe `useBackend()` could take the backend name as a string, and the function would automatically find the corresponding package (if external), load it, and use its backend. That's a better abstraction than requiring the user to specify the backend package name, and the object name. We could also have an `availableBackends()` that lists possible values. I wonder if there are any facilities (e.g., with RStudio) so that it offers completions like `library()`. Another idea is to have an object like `ddr` that is a dynamic list-like database of backends, along with a `use()` generic, so we could have `use(ddr$parallel)`. Just throwing out some ideas.